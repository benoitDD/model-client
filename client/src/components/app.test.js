import App from './app'
import renderer from 'react-test-renderer'
import React from 'react'

test('Test app', () => {
    const component = renderer.create(
        <App/>
    )

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})