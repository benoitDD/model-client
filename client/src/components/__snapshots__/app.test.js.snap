// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app 1`] = `
<p>
  Hello World !!
</p>
`;
